overlord/hookstate/ctlcmd/ctlcmd.go

```diff
@@ -137,7 +137,7 @@ func (f ForbiddenCommandError) Error() string {
 
 // nonRootAllowed lists the commands that can be performed even when snapctl
 // is invoked not by root.
-var nonRootAllowed = []string{"get", "services", "set-health", "is-connected", "system-mode"}
+var nonRootAllowed = []string{"get", "services", "set-health", "is-connected", "system-mode", "model"}
```

Comment:
Is this correct? I need approval for this

Reply:
Looks good to me, this means any user can check the model assertion which should be fine IMHO.